iml-agent/src/action_plugins/ltuer.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+use std::path::PathBuf;
+use tokio_fs as fs;
+
+#[cfg(not(test))]
+static CONFIGURATION_DIR: &str = "/etc/iml";
+#[cfg(test)]
+static CONFIGURATION_DIR: &str = "/tmp/etc/iml";
```

Comment:
It is possible-to avoid working with files at all in tests? 
If not, maybe use something like `mktemp`.?
What about cleaning after tests? Running multiple tests in parallel?

Reply:
Agree

Reply:
In addition, if the file from previous run is left, and the plugin is broken, the test will pass :)

Reply:
If the dir needed to be configurable, you could make a higher-order function that took the dir and returned the plugin. Then you could use something like https://docs.rs/tempfile/3.1.0/tempfile/fn.tempdir.html to ensure there was a unique dir for each run. 

That said, I'm not sure how a closure returned from a HOF would mesh with the current plugin interface (might be ok).

Reply:
I reworked the test.

I didn't use HOF but rather couple of wrappers.